Azure/azure-cli
# [AKS] `az aks mesh`: Add warnings for out of support asm revision in use

**Related command**
`az aks mesh`

**Description**
This change adds warnings to the `az aks mesh` commands for an out of support asm revision in use in the cluster.

**Testing Guide**
`az aks mesh enable -- revision asm-1-18 -g rgName -n clusterName`
This command should print a warning about asm-1-18 being out of support.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
